src/elementary/NucleusID/src/ctor.hpp

```diff
@@ -20,6 +20,17 @@ NucleusID( const std::pair< std::string, int >&& pair ) :
  */
 NucleusID( int z, int a, int l ) : NuclideID( z, a, l ) {}
 
+/**
+ *  @brief Constructor
+ *
+ *  This function throws an invalid_argument exception if invalid data is
+ *  used.
+ *
+ *  @param[in] za   the za number
+ *  @param[in] l    the level number
+ */
+NucleusID( int za, int l ) : NuclideID( za, l ) {}
```

Comment:
How does this thrown an exception? Are you just differing to `NuclideID`? And if so, why wouldn't the other constructor also throw an exception? Do we want to catch/rethrow with additional error messages?

Reply:
std::invalid_exception is thrown when the element number is negative, zero or above 118; when the mass is negative, etc.